js-advanced-functions/fn-exercise-1.js

```diff
@@ -0,0 +1,65 @@
+//1. part one
+let initialData=[200,500,300,1000]
+console.log('INITIAL DATA: ',initialData)
+let newArray=[]
+console.log('Part 1 of the exercise -------------')
+
+const incrementing=(array,padding)=>{
+    for(i=0;i<array.length;i++){
+        array[i]=array[i]+padding
+        // console.log(array)
+    }
+    // for(el of array){
+    //     el=el+padding
+    //     // console.log(el)
+    // }
+    // the two for cycles do the same
+    newArray=array
+    console.log('Incrementing function by 2',newArray)
+}
+incrementing(initialData,padding=2)
+
+let newArray2=[]
+initialData=[200,500,300,1000]
+
+const decrementing=(array,padding)=>{
+    for(i=0;i<array.length;i++){
+        array[i]=array[i]-padding
+        // console.log(array)
+    }
+    newArray2=array
+    console.log('Decrementing function by 2',newArray2)
+}
+decrementing(initialData,padding=2)
+
+//2.part two 
+//HOF
+console.log('Part 2 of the exercise --------------')
+let newArray3=[]
+initialData=[200,500,300,1000]
+
+const operationInc = (array,padding)=>{
```

Comment:
You need to apply the following refactor to this function. 
The incrementing function should receive two parameters, one for a number and the other for the padding.
It should return a value `num + padding`. There is no iterration here. The iterration will be done in the main function

Reply:
@atodoroski Also here, should the code be: 
const operationInc = (number,padding)=>{
  // not sure what to insert here
    return number+padding   
}

Reply:
nothing, the function has just one line. Then you will call this function to apply the increment/decrement operation